test/publish/index.js

```diff
@@ -0,0 +1,85 @@
+const packageJson = require('../../package.json');
+const buildTime = new Date().getTime();
+const path = require('path');
+
+const { spawn } = require('child_process');
+
+const {
+  ECR_REPO,
+  BRANCH,
+  BUILD_NUMBER
+} = process.env;
+
+const getBranchVersions = () => {
+  return BRANCH === 'main' ?
+    [`${packageJson.version}`, 'latest'] :
+    [`${packageJson.version}-${BRANCH}-${BUILD_NUMBER}`];
+};
+
+const getRepo = () => {
+  return ECR_REPO || 'medicmobile';
+};
+
+const getVersions = (release) => {
+  if (BRANCH) {
+    return getBranchVersions(release);
+  }
+  return [`${packageJson.version}-dev.${buildTime}`];
+};
+
+const getImageTags = () => {
+  const versions = getVersions();
+  const tags = versions.map(version => version.replace(/\/|_/g, '-'));
+  return tags.map(tag => `${getRepo()}/cht-upgrade-service:${tag}`);
+};
+
+const dockerCommand = (params) => {
+  const args = [ ...params.filter(param => param).map(param => param.split(' ')) ].flat();
```

Comment:
I think you can now drop this line, because we always pass an array of args now, right?